File: 2026-09/src/day_11/prms_181871/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

int solution(string myString, string pat) {
    int cnt = 0;
    for (int i = 0; i < myString.size(); i++) {
        if (myString.find(pat, i) == i) cnt++;
    }

    return cnt;
}

File: 2026-09/src/day_11/prms_181871/Solution.java
package day_11.prms_181871;

class Solution {
    public int solution(String myString, String pat) {
        int cnt = 0;
        for (int i = 0; i < myString.length(); i++) {
            if (myString.indexOf(pat, i) == i) cnt++;
        }

        return cnt;
    }
}
